### DISCUSSION TITLE
ExceptionList Bug Question

### DISCUSSION THREAD

**Discussion by Mrclopec** [Mr-MIBonk/M.I.B._More-Incredible-Bash#74] [Q&A] [answered]
Good afternoon,

I was reading over the ExceptionList Bug page and was hoping for some guidance before I start modifying. I have an MIB2 unit that was updated from MHI2_US_AU57X_**K0539 (1062)** to MHI2_US_AU57x_KMHI2_US_AU57x_**K3324_1 (0925)**. This was previously done via Teamviewer for the purpose of Carplay activation in 2019. I took notes during the process and distinctly remember the unit reading version MHI2_US_AU57x_**P3237** before finalizing with K3324_1. Do you know if I will likely experience the ExceptionsList Bug if I attempt to patch this system as-is or should I just plan on downgrading the firmware back to another version before doing so?

Thank you for your support.

**Comment by harman-f** [comment, marked as answer]
Your unit could be patched with EL method or also FEC.
In 2019 EL was still used.

As you unit is already upldated and not original anymore you could update to latest FW by using this:
https://mibwiki.one/doc/mhi2_er_au57x_k3663_1-mu1425-aio-R6NuQH8h5p

However, as always if a unit was already patched by an "expert" the is a certain risk, that the unit was "protected" by that person.
Some implemnt functions to delete the unit and other nasty stuff ;)
